reactDashboard/src/App.css

```diff
@@ -1,83 +1,166 @@
-.underHeatmapSelector {
-  border: 1px solid #2a84bb;
-  margin-top: 8px;
-  margin-bottom: 8px;
+.darkmode {
+  background-color: #121212;
 }
 
-.full-width-hr {
-  border: 1px solid #2a84bb;
-  margin: 10px 0;
-  padding: 0;
+.full-screen {
+  background-color: #ffffff;
 }
 
-.full-screen {
-  background-color: White;
+.darkmode .container-fluid {
```

Comment:
Container-fluid is a bootstrap class - is this intended to colour all instances of it, or a specific one? if specific can we use a more specific classname ?

Reply:
Is there a lightmode version of this block?